vms/example/xsvm/api/client.go

```diff
@@ -245,20 +245,23 @@ func (c *client) Message(
 	return resp.Message, resp.Signature, resp.Message.Initialize()
 }
 
-func WaitForAcceptance(
+func AwaitTxAccepted(
```

Comment:
this is a bit uncommon go function signature. I would expect it to look like
```golang
func (c Client) AwaitTxAccepted(ctx, address, nonce, freq, options)
```

Reply:
Client is an interface